portal/src/app/search/CustomListDialog.jsx

```diff
@@ -83,10 +85,11 @@ class CustomListDialog extends React.Component {
 
     setText = (value) => {
```

Comment:
Not really a review comment, just a curiosity based on my own ES6 learning - Why use the arrow function and assign it to a name like `setText`? Is it done to avoid stomping on `this`?